internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -318,6 +330,32 @@ Resources:
                 Action:
                   - s3:PutObject
                 Resource: !Sub arn:aws:s3:::${Bucket}/*
+        - PolicyName: CacheInvalidation
```

Comment:
Do we have any reference for this? How do we make sure this is the minimum required policy? especially for `Resource: "*"` for the first statement

Reply:
The third example in https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/security_iam_id-based-policy-examples.html#security_iam_id-based-policy-examples-aws-managed

Reply:
Maybe we add the shortened source URL as a comment?

Reply:
If we're concerned about the permissiveness, should we add the copilot tag condition?

Reply:
Im not sure if tag works for this. Maybe test it out and we should def. do if possible.

Reply:
added the tags condition